packages/api-client/src/types/types.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
Because this file is listed in eslint ignore but it's not working on my local and doing this is the only way I can commit it.

Reply:
If this is the only issue you are having with eslint you can commit with `--no-verify` rule. Because if we merge your current code it will make other people working on this project have problems with this file that are caused by a local linter.

Reply:
There were a few no-use-before-define issues that although didn't really matter were throwing a linting error. Changed the order and that fixed it.